test/server/cards/01_SOR/events/RebelAssault.spec.js

```diff
@@ -0,0 +1,31 @@
+describe('Rebel Assault', function() {
+    integration(function() {
+        describe('Rebel Assault\'s ability', function() {
+            beforeEach(function () {
+                this.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['rebel-assault'],
+                        groundArena: ['pyke-sentinel', 'battlefield-marine'],
+                        spaceArena: ['green-squadron-awing']
+                    },
+                    player2: { }
```

Comment:
Can you add an enemy Rebel unit here to confirm that Rebel Assault doesn't somehow trigger it to attack, and add a rebel leader like Leia to confirm that leaders are selectable